engine/src/test/java/org/camunda/bpm/engine/test/api/optimize/GetHistoricVariableUpdatesForOptimizeTest.java

```diff
@@ -277,57 +337,20 @@ public void fetchOnlyVariableUpdates() {
     runtimeService.startProcessInstanceByKey("process");
 
     Task task = taskService.createTaskQuery().singleResult();
-    Map<String, String> formFields = new HashMap<>();
+    Map<String, Object> formFields = new HashMap<>();
     formFields.put("var", "foo");
-    engineRule.getFormService().submitTaskFormData(task.getId(), formFields);
+    engineRule.getFormService().submitTaskForm(task.getId(), formFields);
     long detailCount = engineRule.getHistoryService().createHistoricDetailQuery().count();
     assertThat(detailCount).isEqualTo(2L); // variable update + form property
 
     // when
     List<HistoricVariableUpdate> variableUpdates =
-      optimizeService.getHistoricVariableUpdates(pastDate(), null, 10);
+      optimizeService.getHistoricVariableUpdates(pastDate(), null, false, 10);
 
     // then
     assertThat(variableUpdates.size()).isEqualTo(1);
   }
 
-  @Test
-  public void getHistoricVariableByteArrayUpdates() {
```

Comment:
Please correct me if I am wrong, but there is no test case yet (and has never been I guess) that tests that variables of types `BYTES` and `FILE` are never fetched (no matter if `excludeObjectValues` is true or false). Would you mind adding such test cases to also strengthen the integrity of the adjusted condition in `shouldFetchValue`?

Reply:
Good catch will do!